setup.py

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env python
-import io
+import codecs
+import os
 from setuptools import setup
 
 
-def read(fname):
-    with io.open(fname, encoding="utf8") as fp:
-        content = fp.read()
-    return content
+# From https://packaging.python.org/en/latest/guides/single-sourcing-package-version/
+def read(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
+    with codecs.open(os.path.join(here, rel_path), 'r') as fp:
+        return fp.read()
+
+
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith('__version__'):
+            delim = '"' if '"' in line else "'"
+            return line.split(delim)[1]
+    else:
+        raise RuntimeError("Unable to find version string.")
 
 
 setup(
     name="judyz-cffi",
-    version="0.8.5",
+    version=get_version("judyz_cffi/__init__.py"),
```

Comment:
Are you familiar with [setuptools-scm](https://pypi.org/project/setuptools-scm/) ? I like this tool to simplify the question of extracting the version without having to put it in a commit. It extract version number from git, so if you're on a tag, the version number will be the tag name.